examples/meta/generator/targets/csharp.json

```diff
@@ -32,6 +32,19 @@
         "LongRealMatrix": "var $name = new double[$arguments]",
         "ComplexMatrix": "var $name = new Complex[$arguments]"
     },
+    "ListInit": {
+        "BoolVector": "var $name = new bool[] {$arguments}",
+        "CharVector": "var $name = new char[] {$arguments}",
+        "ByteVector": "var $name = new byte[] {$arguments}",
+        "WordVector": "var $name = new ushort[] {$arguments}",
+        "IntVector": "var $name = new int[] {$arguments}",
+        "LongIntVector": "var $name = new long[] {$arguments}",
+        "ULongIntVector": "var $name = new ulongint[] {$arguments}",
+        "ShortRealVector": "var $name = new float[] {$arguments}",
```

Comment:
need to fix c#, floats need to have a 'f' suffix, i.e. 1.0, should become 1.0f ...